apps/voting/app/src/components/Vote.js

```diff
@@ -0,0 +1,274 @@
+  useTheme,
+} from '@aragon/ui'
+import { useAppState } from '@aragon/api-react'
+import LocalIdentityBadge from './LocalIdentityBadge/LocalIdentityBadge'
+import VoteText from './VoteText'
+import VoteStatus from './VoteStatus'
+import SummaryBar from './SummaryBar'
+import SummaryRows from './SummaryRows'
+import VoteActions from './VoteActions'
+import { getQuorumProgress } from '../vote-utils'
+import { VOTE_NAY, VOTE_YEA } from '../vote-types'
+import { percentageList, round, safeDiv } from '../math-utils'
+import { formatDate } from '../utils'
+
+const DEFAULT_DESCRIPTION =
+  'No additional description has been provided for this proposal.'
+const SEPARATOR = "for '"
+const EXECUTING = ' , executing'
+
+function Vote({ vote, onVote, onExecute }) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const { tokenSymbol } = useAppState()
+  const { data, numData, voteId, connectedAccountVote } = vote
+  const { minAcceptQuorum, supportRequired, yea, nay } = numData
+  const { creator, endDate, open, metadata, description } = data
+  const text = description || metadata
+  const [rawTitle, extra = DEFAULT_DESCRIPTION] =
+    text.indexOf(SEPARATOR) > -1 ? text.split(SEPARATOR) : [text]
+  const [title] = rawTitle.indexOf(EXECUTING)
+    ? rawTitle.split(EXECUTING)
+    : [title]
+  const quorumProgress = getQuorumProgress(vote)
+  const totalVotes = yea + nay
+  const votesYeaVotersSize = safeDiv(yea, totalVotes)
+  const votesNayVotersSize = safeDiv(nay, totalVotes)
+  const youVoted =
+    connectedAccountVote === VOTE_YEA || connectedAccountVote === VOTE_NAY
+  const handleVoteNo = useCallback(() => {
+    onVote(voteId, VOTE_NAY)
+  }, [onVote, voteId])
+  const handleVoteYes = useCallback(() => {
+    onVote(voteId, VOTE_YEA)
+  }, [onVote, voteId])
+
+  const handleExecute = useCallback(() => {
+    onExecute(voteId)
+  }, [onExecute, voteId])
+  const [yeaPct, nayPct] = percentageList(
+    [votesYeaVotersSize, votesNayVotersSize],
+    2
+  )
+
+  if (!vote) {
+    return null
+  }
+
+  return (
+    <Split
+      primary={
+        <Box>
+          <div
+            css={`
+              display: flex;
+              justify-content: space-between;
+            `}
+          >
+            <Tag mode="app">App Badge</Tag>
+            {youVoted && (
+              <div
+                css={`
+                  display: inline-grid;
+                  grid-template-columns: auto auto;
+                  grid-gap: ${0.5 * GU}px;
+                  align-items: center;
+                  justify-content: center;
+                  height: 20px;
+                  width: auto;
+                  border-radius: 100px;
+                  padding: 0 ${1 * GU}px;
+                  background: ${theme.infoSurface.alpha(0.08)};
+                  color: ${theme.info};
+                  ${textStyle('label2')};
+                `}
+              >
+                <IconCheck size="tiny" /> Voted
+              </div>
```

Comment:
@dizzypaty Should this be a `<Tag size="compact">`?

Currently using that displays:

<img width="96" alt="Screen Shot 2019-08-09 at 3 10 53 PM" src="https://user-images.githubusercontent.com/4166642/62781316-00b32500-bab8-11e9-9336-458c756ed29d.png">

Which is quite different than the [design](https://www.figma.com/file/O09HJRCFDwDTni4VE3tiRvwI/Voting?node-id=375%3A866).